app/src/components/main/attributeList/editableAttribute/attributeEditor/attributeEditor.jsx

```diff
@@ -52,6 +52,9 @@ const attributeValueValidator = composeBoundValidators([
   commonValidators.requiredField,
   bindMessageToValidator(validate.attributeValue, 'attributeValueLengthHint'),
 ]);
+const attributeFilterValueValidator = composeBoundValidators([
+  bindMessageToValidator(validate.attributesFilterValue, 'attributeFilterValueLengthHint'),
+]);
```

Comment:
The `composeBoundValidators` not needed if you use only one validator.
```javascript
const attributeFilterValueValidator = bindMessageToValidator(validate.attributesFilterValue, 'attributeFilterValueLengthHint');
```

Reply:
As `attributeFilterValueValidator` stored here in the commom `AttributeEditor` component and you use it as validator for non required value, so may be rename it f.e. to `nonRequiredAttibuteValueValidator`?
This will help to clearly reuse non required validation for attribute key in other places (not just in filters).